### PR TITLE
Homework #7

### PR DESCRIPTION
**Pull request by AndryNosach**
Done:
1. Added LiquidBase to the project including changelog-db.xml with db structure.
2. Added Spring DI.

### CODE REVIEW

File: Nosach/web-quiz-project/src/main/java/dao/QuizDAOImpl.java [hunk 1]
```diff
@@ -4,16 +4,22 @@
...
 import java.util.List;
 
+@Component
 public class QuizDAOImpl implements QuizDAO {
 
     private static final Logger logger = Logger.getLogger(QuizDAOImpl.class);
 
+    @Autowired
+    SubjectService ss;
+
```
Comment (dinamio) [inline Nosach/web-quiz-project/src/main/java/dao/QuizDAOImpl.java:22]: better to name subjectService, for more readability.

File: Nosach/web-quiz-project/src/main/java/dao/UserDAOImpl.java [hunk 2]
```diff
@@ -3,16 +3,23 @@
...
 import java.sql.PreparedStatement;
 import java.sql.ResultSet;
 import java.sql.SQLException;
 
+@Component
 public class UserDAOImpl implements UserDAO {
 
     private static final Logger logger = Logger.getLogger(UserDAOImpl.class);
 
+    public UserDAOImpl() {
```
Comment (dinamio) [inline Nosach/web-quiz-project/src/main/java/dao/UserDAOImpl.java:20]: This constructor is autogenerated, so no need in it

**Review by dinamio — APPROVED** [review]
Very good! See minor suggestions

File: Nosach/web-quiz-project/src/main/java/dao/QuizDAOImpl.java (hunk 1 above)
Comment (AndryNosach) [reply Nosach/web-quiz-project/src/main/java/dao/QuizDAOImpl.java:22]: Ok, will do

File: Nosach/web-quiz-project/src/main/java/dao/UserDAOImpl.java (hunk 2 above)
Comment (AndryNosach) [reply Nosach/web-quiz-project/src/main/java/dao/UserDAOImpl.java:20]: This was for debugging bean, forgot to delet. Will do.

**Pull request merged**